client/src/helpers/fhir/fhir_data_manager.ts

```diff
@@ -55,11 +55,7 @@ export class FhirDataManager {
                 }
             ],
             gender: options.gender,
-            generalPractitioner: [
-                {
-                    reference: `Practitioner/${options.practitionerId}`
-                }
-            ],
+            generalPractitioner: [],
```

Comment:
No longer need to associate the patient with the Practitioner?  Or is it done somewhere else?

Reply:
We do it, but later on since we select the practitioner on page 3, but we need to be able to save a patient on page 1.